assets/model_monitoring/components/src/model_monitor_output_metrics/entities/signals/data_drift_signal.py

```diff
@@ -149,11 +149,12 @@ def _build_feature_metrics(
 
             # Add the feature importance metrics
             if len(featureimportance_dictionary) != 0:
-                feature_importance_metric = {
-                    "metricName": "BaselineFeatureImportance",
-                    "metricValue": featureimportance_dictionary[feature_name],
-                }
-                output[feature_name].metrics.append(feature_importance_metric)
+                if not any('BaselineFeatureImportance' in d.values() for d in output[feature_name].metrics):
```

Comment:
This can be a later refactor: I feel like because this for loop in line 102 is for metrics level, but the feature importance is in feature level, instead of putting this logic inside this loop, it is better to do a separate one for output.

i.e. something like:
`if len(featureimportance_dictionary) != 0:
         for feature in output:
               output[feature].metrics.append(feature_importance_metric)`